planner/core/physical_plan_test.go

```diff
@@ -189,6 +189,15 @@ func (s *testPlanSuite) TestDAGPlanBuilderSimpleCase(c *C) {
 			sql:  "select * from (select * from t use index() order by b) t left join t t1 on t.a=t1.a limit 10",
 			best: "IndexJoin{TableReader(Table(t)->TopN([test.t.b],0,10))->TopN([test.t.b],0,10)->TableReader(Table(t))}(test.t.a,t1.a)->Limit",
 		},
+		// Test nest different size order by.
```

Comment:
How about "Test embedded ORDER BY which imposes on different number of columns than outer query"?